docs/source/start.ipynb

```diff
@@ -58,15 +58,15 @@
    "source": [
     "import woodwork as ww\n",
     "\n",
-    "df.ww.init(name=\"retail\", make_index=True, index=\"order_product_id\")\n",
+    "df.ww.init(name=\"retail\")\n",
```

Comment:
Later on in the docs, I believe we try and set the index to `order_product_id` which will now fail. I believe there is no unique column on this dataframe, so in order to show setting the index, you may have to manually add the index to `df` here.